src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +49,30 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get the historical price data for a given stock symbol.
+   *
+   * @param symbol stock symbol to get historical data for.
+   * @param range specified time range of the request.
+   * @param date specified date in the format YYYYMMDD.
```

Comment:
Would you want to do anything to enforce that the data is specified in that format?

Reply:
I've transferred the logic into IexService and have added some format checks for symbol, range, and date.